source/features/archive-forks-link.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'dom-chef';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../feature-manager';
+import observe from '../helpers/selector-observer';
+import {buildRepoURL} from '../github-helpers';
+
+function addLinkToBanner(banner: HTMLElement): void {
+	if (banner.lastChild!.textContent!.includes('repository has been archived')) {
```

Comment:
@kidonng I just realized we have this in the detections, as well as isPublicRepo which I had to drop from `useful-forks`.

Would you mind linting this in your PR?

```suggestion
	if (isPublicRepo() && isArchivedRepo()) {
```

Reply:
Done in [`66038a2` (#6488)](https://github.com/refined-github/refined-github/pull/6488/commits/66038a22e503958b9cfe9d761a10df79cd59754f)